test/terms.jl

```diff
@@ -7,7 +7,15 @@ mimestring(x) = mimestring(MIME"text/plain", x)
 
 @testset "terms" begin
     using TuringGLM:
```

Comment:
If you do implicit imports in runtests, then you can also test which methods are exported.

Also, it's usually best to do all the code loading inside the main file (runtests or src/TuringGLM) to avoid duplicate loading of symbols (which isn't an issue per se, but more typing). See https://github.com/TuringLang/Turing.jl/pull/1548 to get an idea of how much duplication one can end up with if loading is done in multiple files.

Reply:
I don't understand. I am testing "private" functions (non-exported). Are you sugestting just to load the the `using` statements at the `runtests.jl` or to not `using: YADA` anything and do all with `TuringGLM.YADA`?

Reply:
Most people do indeed `TuringGLM.YADA` in their tests and sometimes abbreviate the module name to something like `T`. So,
```
const T = TuringGLM

@test T.YADA(...) = ...
```